MemoryGame-1B/Views/NewGame.xaml

```diff
@@ -8,55 +8,98 @@
       d:DesignHeight="450" d:DesignWidth="800"
       Title="NewGame">
     <Grid>
+        <Grid.Background>
+            <ImageBrush ImageSource="background.jpg"/>
+        </Grid.Background>
         <Grid Margin="10">
             <Grid.ColumnDefinitions>
                 <ColumnDefinition Width="240*"/>
                 <ColumnDefinition Width="120*"/>
             </Grid.ColumnDefinitions>
             <Grid.RowDefinitions>
                 <RowDefinition Height="120*"></RowDefinition>
+                <RowDefinition Height="30*"></RowDefinition>
             </Grid.RowDefinitions>
             <Grid x:Name="Grid" Column="0">
 
+            </Grid>
+            <Grid x:Name="imgBottomZombies" Grid.Row="2" Column="0" >
+                <Image Source="..\Images\Placeholder.png" Stretch="Fill"/>
             </Grid>
             <Grid x:Name="Score" Column="1">
                 <Grid.RowDefinitions>
                     <RowDefinition Height="60*"></RowDefinition>
                     <RowDefinition Height="60*"></RowDefinition>
+                    <RowDefinition Height="60*"></RowDefinition>
                 </Grid.RowDefinitions>
                 <Grid Column="1" Row="0">
                     <Grid.RowDefinitions>
+                        <RowDefinition Height="20*"></RowDefinition>
                         <RowDefinition Height="60*"></RowDefinition>
                         <RowDefinition Height="60*"></RowDefinition>
-                        <RowDefinition Height="60*"></RowDefinition>
-                        <RowDefinition Height="60*"></RowDefinition>
+                        <RowDefinition Height="20*"></RowDefinition>
                     </Grid.RowDefinitions>
+
                     <Grid Column="1"  Row="1">
                         <Grid.RowDefinitions>
-                            <RowDefinition Height="30*"></RowDefinition>
-                            <RowDefinition Height="30*"></RowDefinition>
+                            <RowDefinition  Height="40*"></RowDefinition>
+                            <RowDefinition Height="40*"></RowDefinition>
                         </Grid.RowDefinitions>
-                        <TextBlock Grid.Row="0" Margin="30 0 0 0">Playername1</TextBlock>
-                        <TextBlock Grid.Row="1" Margin="30 0 0 0">score 0</TextBlock>
+                        <Canvas Width="{Binding ElementName=gd, Path=ActualWidth}"
```

Comment:
what is gd here please use a more descriptive name like ElementName

Reply:
Removed doesn't work